utkdigitalinitiatives/utk_isl_xml_forms
# Complete CRUD operations for TRAC-1036.

**JIRA Ticket**: [TRAC-1036](https://jira.lib.utk.edu/browse/TRAC-1036)

# What does this Pull Request do?
Adds CRUD operations for Law form

# What's new?
This should create, read, update, and delete all the correct nodes for the law metadata mapping listed here: 

https://docs.google.com/spreadsheets/d/1JOXRWpwz0nAnWitHmGyLbfHTTpiBXXXAeqfSeMAJhPo/edit#gid=0

# How should this be tested?

* Test the entire workflow:
    * Import the Form
    * Associate it with a content model
    * Create a new record and select the newly associated form
    * Edit that record to verify proper CRUD behavior

# Interested parties
@CanOfBees
